test/touchevents.test.tsx

```diff
@@ -160,15 +160,15 @@ describe('TouchEventBoundary._onTouchStart', () => {
     expect(addBreadcrumb).toBeCalledWith({
       category: defaultProps.breadcrumbCategory,
       data: {
-        componentTree: ['Styled(View2)', 'Styled(View)'],
+        path: [{ name: 'Styled(View)' }],
```

Comment:
`'Styled(View2)'` is not in the path because it has a label which is ignored, was this intentional?

Before when label was ignored the code took the next value, but now when ignored label is read it will drop the value from the path.

Reply:
I thought the original behavior was a bug - since you specified you want to ignore something, I'd expect the element to be ignored, not just the label (and element taken in path). I can make it work the original way, of course, if you think it's necessary.

Reply:
No, I also think it was a bug, the previous behaviour seems very unexpected to me as well!

I just wanted to make sure we are making conscious decision to change this.